js/web/docs/webnn-operators.md

```diff
@@ -0,0 +1,92 @@
+| LayerNormalization | ai.onnx(7-16, 17+) | layerNormalization | ✗ | ✓ | |
+| LeakyRelu | ai.onnx(7-15, 16+) | leakyRelu | ✓ | ✓ | |
+| Less | ai.onnx(7-8, 9-12, 13+) | lesser | ✗ | ✓ | |
+| LessOrEqual | ai.onnx(12-15, 16+) | lesserOrEqual | ✗ | ✓ | |
+| Log | ai.onnx(7-12, 13+) | log | ✗ | ✓ | |
+| LpPool | ai.onnx(7-10, 11-17, 18+) | l2Pool2d | ✗ | ✓ | Only supports 4-D input, 2-D 'kernel_shape', 'p' value is 2 |
+| MatMul | ai.onnx(7-8, 9-12, 13+) | matmul | ✓ | ✓ | WebNN CPU doesn't support broadcasting for MatMul |
+| Max | ai.onnx(7, 8-11, 12, 13+) | max | ✓ | ✓ | |
+| MaxPool | ai.onnx(7, 8-9, 10, 11, 12+) | maxPool2d | ✓ | ✓ | Only supports 4-D input, 2-D 'kernel_shape', 'storage_order' != 1, one output |
+| Min | ai.onnx(7, 8-11, 12, 13+) | min | ✓ | ✓ | |
+| Mul | ai.onnx(7-12, 13, 14+) | mul | ✓ | ✓ | |
+| Neg | ai.onnx(7-12, 13+) | neg | ✓ | ✓ | |
+| Not | ai.onnx(7+) | logicalnot | ✗ | ✓ | |
+| Pad | ai.onnx(7-10, 11-12, 13-17, 18, 19-20, 21+) | pad | ✓ | ✓ | modes == 'wrap' is not supported |
+| Pow | ai.onnx(7-11, 12, 13-14, 15+) | pow | ✗ | ✓ | |
+| PRelu | ai.onnx(7-8, 9-15, 16+) | prelu | ✓ | ✓ | WebNN CPU restricts slope to be a static value |
+| Reciprocal | ai.onnx(7-12, 13+) | reciprocal | ✗ | ✓ | |
+| ReduceL1 | ai.onnx(7-10, 11-12, 13-17, 18+) | reduceL1 | ✗ | ✓ | Input 'axes' if present should be a constant |
+| ReduceL2 | ai.onnx(7-10, 11-12, 13-17, 18+) | reduceL2 | ✗ | ✓ | Input 'axes' if present should be a constant |
+| ReduceLogSum| ai.onnx(7-10, 11-12, 13-17, 18+) | reduceLogSum| ✗ | ✓ | Input 'axes' if present should be a constant |
+| ReduceLogSumExp | ai.onnx(7-10, 11-12, 13-17, 18+) | reduceLogSumExp | ✗ | ✓ | Input 'axes' if present should be a constant |
+| ReduceMax | ai.onnx(7-10, 11, 12, 13-17, 18-19, 20+) | reduceMax | ✗ | ✓ | Input 'axes' if present should be a constant |
+| ReduceMean | ai.onnx(7-10, 11-12, 13-17, 18+) | reduceMean | ✓ | ✓ | Input 'axes' if present should be a constant |
+| ReduceMin | ai.onnx(7-10, 11, 12, 13-17, 18-19, 20+) | reduceMin | ✗ | ✓ | Input 'axes' if present should be a constant |
+| ReduceProd | ai.onnx(7-10, 11-12, 13-17, 18+) | reduceProduct | ✗ | ✓ | Input 'axes' if present should be a constant |
+| ReduceSum | ai.onnx(7-10, 11-12, 13+) | reduceSum | ✗ | ✓ | Input 'axes' if present should be a constant |
+| ReduceSumSquare | ai.onnx(7-10, 11-12, 13-17, 18+) | reduceSumSquare | ✗ | ✓ | Input 'axes' if present should be a constant |
+| Relu | ai.onnx(7-12, 13, 14+) | relu | ✓ | ✓ | |
+| Reshape | ai.onnx(7-12, 13, 14-18, 19-20, 21+) | reshape | ✓ | ✓ | Input 'shape' should be a constant, 0 dimension value in 'shape' is not supported |
+| Resize | ai.onnx(11-12, 13-17, 18, 19+) | resample2d | ✓ | ✓ | Only supports 4-D input, exclude_outside != 0, input 'scales' and 'sizes' if present must be a constant, WebNN CPU backend only supports 'linear' mode, WebNN GPU backend only supports 'linear' and 'nearest' modes |
```

Comment:
For WebNN CPU, the axes must be {1, 2}.

Reply:
This is determined by the layout, if it is NHWC, we set axes to `{1, 2}` otherwise `{2, 3}`

Reply:
Ack.